src/lib.rs

```diff
@@ -38,6 +38,8 @@ pub type Request = http_types::Request;
 /// An HTTP Response type with a streaming body.
 pub type Response = http_types::Response;
 
+pub use http_types;
```

Comment:
not sure if I should remove this — my original plan was to remove the direct dependency from surf and only use the re-export from http-client to make updates easier in the future, but that change was far too sprawling

Reply:
That makes sense. Happy to keep things as they are now. Something we should still explore is whether we can deprecate http-client as well.